functional-test/src/main/java/org/zanata/page/projects/projectsettings/ProjectLanguagesTab.java

```diff
@@ -111,37 +119,24 @@ public ProjectLanguagesTab enterSearchLanguage(String languageQuery) {
      * Add a language to the languages list.
      * Assumes that the search string has already been entered.
```

Comment:
Does this comment about the search string still count, now that this does not use an autocomplete component?